viz.yaml

```diff
@@ -298,7 +298,7 @@ process:
     scripts: scripts/process/process.precip_colors.R
     reader: rds
     depends: ["precip-colors"]
-    stepSize: 6
+    stepSize: 3
```

Comment:
Nice! Thinking we may want a few more labels on this color scale since it's such a broad range?

Reply:
agreed.